docs/output.md

```diff
@@ -6,36 +6,58 @@ This document describes the output produced by the pipeline. Most of the plots a
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
-- [FastQC](#fastqc) - Raw read QC
+- [Mutect2](#mutect2) - Variant calls from Mutect2
+- [GenomicsDB](#genomicsdb) - GenomicsDB output
+- [CreateSomaticPanelOfNormals](#createsomaticpanelofnormals) - CreateSomaticPanelOfNormals output
 - [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
 - [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
 
-### FastQC
+### Mutect2
 
 <details markdown="1">
 <summary>Output files</summary>
 
-- `fastqc/`
-  - `*_fastqc.html`: FastQC report containing quality metrics.
-  - `*_fastqc.zip`: Zip archive containing the FastQC report, tab-delimited data file and plot images.
+- `mutect2/${sample}`
+  - `*.vcf.gz`: Variant calling results
+  - `*.vcf.gz.tbi`: Index matching the vcf file
 
 </details>
 
-[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your sequenced reads. It provides information about the quality score distribution across your reads, per base sequence content (%A/T/G/C), adapter contamination and overrepresented sequences. For further reading and documentation see the [FastQC help pages](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+### GenomicsDB
 
-![MultiQC - FastQC sequence counts plot](images/mqc_fastqc_counts.png)
+<details markdown="1">
+<summary>Output files</summary>
 
-![MultiQC - FastQC mean quality scores plot](images/mqc_fastqc_quality.png)
+- `genomicsdbimport/${sample}`
+  - various output files
+
+</details>
+
+### CreateSomaticPanelOfNormals
+
+<details markdown="1">
+<summary>Output files</summary>
 
-![MultiQC - FastQC adapter content plot](images/mqc_fastqc_adapter.png)
+- `mutect2_panelofnormals/`
+  - `*.vcf.gz`: Panel of normals
+  - `*.vcf.gz.tbi`: Index matching the vcf file
 
-> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
+</details>
+
+### CNVKit
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `cnvkit/`
+  - `reference.cnn`: Reference "panel of normals",
+  - `${sample}.{anti,}target.cnn`: Target and antitagret  reference files
```

Comment:
```suggestion
  - `${sample}.{anti,}target.cnn`: Target and antitarget reference files
```

Reply:
Does it really return the antitarget files with the batch command?

Reply:
yes